### PR TITLE
Updated README and prettyfied Python files

### PR DESCRIPTION
**Pull request by justin-wong-ce**
Examples outputs in the README is updated to the latest adapter and cas-bacnet-stack versions, and instructions on where to place the required DLL file.

Python files are prettyfied and print statements are updated to use f-strings,

### CODE REVIEW

**Comment by justin-wong-ce** [comment]
Added NetworkPort object and Send I-Am usage to this example

File: README.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ A basic BACnet IP server example written with Python 3.7 using the [CAS BACnet S
...
+FYI: CAS BACnet Stack version: 3.25.0.0
+FYI: CAS BACnet Stack python adapter version: 0.0.4
 FYI: Connecting UDP Resource to port=[47808]
-FYI: Local IP address:  192.168.2.189
+FYI: Local IP address:  192.168.1.159
 FYI: Registering the Callback Functions with the CAS BACnet Stack
 FYI: Setting up server Device. device.instance=[389001]
 FYI: Adding AnalogInput. AnalogInput.instance=[0]
-FYI: Adding BinaryInput. BinaryInput.instance=[3]
+FYI: Adding BinaryInput. BinaryInput.instance=[db['binaryInput']['instance']]
```
Comment (ssmethurst) [inline README.md]: This should be a number I think this variable is  being rendered as text

**Comment by ssmethurst** [comment]
These variables in the database must be updated when the example starts up. 

"ipAddress": [192, 168, 1, 159],
"ipDefaultGateway": [192, 168, 1, 23],
"ipDnsServer": [1, 2, 3, 4, 5],
"ipSubnetMask": [255, 255, 255, 0],

For example in the main() function, you would query the hardware to find out what the local IP address is, then update the database with that Ip address. This will allow this example to work on anyone's computer.

**Comment by justin-wong-ce** [comment]
Fixed bad text rendering for BinaryInput and added query to get system's DNS server addresses

**Pull request merged**